src/types/identity/index.ts

```diff
@@ -11,10 +12,16 @@ export type CreateIdentityOptions = {
 
 export type NewIdentityRequest = {
   strategy: IdentityStrategy;
-  messageSignature: string;
   options: CreateIdentityOptions;
+  walletType: EWallet;
+  messageSignature?: string;
 };
 
+export enum EWallet {
+  ETH_WALLET,
+  CRYPT_KEEPER,
```

Comment:
```suggestion
  EXTERNAL_WALLET,
  CK_WALLET,
```

Reply:
I don't agree we need to have `EXTERNAL_WALLET` here because we can have multiple wallets which have different implementations (for instance, ethereum and solana).

Reply:
I'd keep it as-is.
```
export enum EWallet {
  ETH_WALLET,
  CRYPT_KEEPER_WALLET,
}
```